src/Countries/Ecuador.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use Spatie\Holidays\Concerns\Observable;
+use Spatie\Holidays\Concerns\Translatable;
+use Spatie\Holidays\Contracts\HasTranslations;
+
+class Ecuador extends Country implements HasTranslations
+{
+    use Observable;
+    use Translatable;
+
+    public function countryCode(): string
+    {
+        return 'ec';
+    }
+
+    public function defaultLocale(): string
+    {
+        return 'en';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'New Year\'s Day' => '01-01',
+        ], $this->variableHolidays($year));
+    }
+
+    public function nearestDay(int $year, int $month, int $day): CarbonImmutable
+    {
+        $date = CarbonImmutable::createFromDate($year, $month, $day);
+
+        if($date->is('Tuesday') || $date->is('Saturday')) {
+            return $date->subDay();
+        }
+
+        if($date->is('Sunday')) {
+            return $date->addDay();
+        }
+
+        if($date->is('Wednesday') || $date->is('Thursday')) {
+            return $date->next(CarbonImmutable::FRIDAY);
+        }
+
+        return $date;
+    }
+
+    public function getChristmasHoliday(int $year): CarbonImmutable
+    {
+        if($year === 2022) {
```

Comment:
Why is only 2022 an observed christmas?

Reply:
Because only in 2022 Christmas was moved from Sunday to Monday, and also I made a mistake using `observedChristmasDay` because it changes from Sunday to Tuesday, so I changed to use `sundayToNextMonday` [d693203](https://github.com/spatie/holidays/pull/218/commits/d693203c0a7695b1c2824afdf56bf8a6b421e8d1)